docs/layouts/shortcodes/endpoint-description.md

```diff
@@ -35,9 +35,13 @@ Or the `redis-persistent` type for persistent Redis:
 
 Persistent Redis requires a disk to store data.
 {{ else if eq $type "network-storage" }}
-You can define `<SERVICE_NAME>` as you like, but it should not include underscores (`_`).
+You can define `<SERVICE_NAME>` as you like, but it shouldn't include underscores (`_`).
 {{ end }}
 
+Note that if you later change the name, it's treated as an entirely new service.
+This removes all data from your service.
+Always backup your data before changing the service.
```

Comment:
:pick: :arrow_forward: 
```suggestion
Always backup your data before changing the service name.
```

Reply:
I'd thought about that, but then I thought that it's true for any part of the service, so I left out `name` intentionally as good advice for any change. :shrug:

Reply:
Fair point, better to be overly precautious indeed!